src/views/RichWorkspace.vue

```diff
@@ -291,6 +291,25 @@ export default {
 		margin: 0;
 	}
 
+	#rich-workspace:deep(.table-wrapper .content) {
```

Comment:
Do you know why this only affects the rich workspaces?

Reply:
> Do you know why this only affects the rich workspaces?

Looks like it's actually some styling from Files affecting it - particularly this part: https://github.com/nextcloud/server/blob/6bced389b20e676babbe193c7fbdf77f66433c3a/apps/files/src/components/FilesListVirtual.vue#L409

Reply:
All right, that seems reasonable then. While ideally the files app would not have such aggressive rules I also cannot think of an elegant approach to limit the scope properly. Fine with this one here then from my side 👍